### PR TITLE
feat(error): add error response interface

### PR DESCRIPTION
**Pull request by mkucmus**
It adds a common interface of response when the exception is thrown on API side.

### CODE REVIEW

File: packages/shopware-6-client/src/interfaces/response/Error.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    line: number;
+    function: string;
+    class: string;
+    type: string;
+    args: Array<any>;
+  }>;
+}
+
+export interface ErrorResponse {
+  errors: Array<Error>;
```
Comment (patzick) [inline packages/shopware-6-client/src/interfaces/response/Error.ts]:
```suggestion
  errors: Error[];
```

File: packages/shopware-6-client/src/interfaces/response/Error.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+interface Error {
+  status: string;
+  code: string;
+  title: string;
+  detail: string;
+  meta: any;
+  trace?: Array<{
```
Comment (patzick) [inline packages/shopware-6-client/src/interfaces/response/Error.ts]: could you move it to separate interface?
```suggestion
  trace?: ErrorTrace[];
```

File: packages/shopware-6-client/src/interfaces/response/Error.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    file: string;
+    line: number;
+    function: string;
+    class: string;
+    type: string;
+    args: Array<any>;
```
Comment (patzick) [inline packages/shopware-6-client/src/interfaces/response/Error.ts]: isn't it a string array?
```suggestion
    args: any[];
```

**Comment by mkucmus** [comment]
Fixes were added. All checks are green :+1:

**Pull request merged**